Add `osu!catch World Cup 2024: Finals Recap` news post (#11659)
ppy/osu-wiki: news/2024/2024-06-27-osu-catch-world-cup-2024-finals-recap.md

---
layout: post
title: "osu!catch World Cup 2024: Finals Recap"
date: 2024-06-27 20:00:00 +0000
---

We have our podium finisher of this years world cup! Let's recap the Finals weekend.

![](/wiki/Tournaments/CWC/2024/img/cwc2024-banner.jpg)

Finals week is notoriously a very exciting weekend as we finally know which three teams are on the podium for this years osu!catch world cup.

## Mappool showcase

<style>
    .osu-md--news .audio-player {
        max-width: inherit;
    }
</style>

[**Download the pool here! (199 MB)**](https://osu.ppy.sh/beatmaps/packs/P251)

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2167854236&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
</div>

> The mappool has reached a difficulty where a full-combo on any of the maps is becoming challenging for some and impossible for many. We saw a handfull of FCs on [HD3](https://osu.ppy.sh/beatmapsets/2198816#fruits/4653147), [HR1](https://osu.ppy.sh/beatmapsets/2198791#fruits/4653099) and [HR3](https://osu.ppy.sh/beatmapsets/2198812#fruits/4653138).

Reviewer: `full-combo` can be `full combo`?

---
layout: post
title: "osu!catch World Cup 2024: Finals Recap"
date: 2024-06-27 20:00:00 +0000
---

> We have our podium finisher of this years world cup! Let's recap the Finals weekend.

Reviewer: ```suggestion
We have our podium finisher of this year's World Cup! Let's recap the Finals weekend.
```

Reviewer 2: again "world cup" doesn't need to be capitalised on its own

Reviewer: fixed

## Mappool showcase

<style>
    .osu-md--news .audio-player {
        max-width: inherit;
    }
</style>

[**Download the pool here! (199 MB)**](https://osu.ppy.sh/beatmaps/packs/P251)

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2167854236&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
</div>

The mappool has reached a difficulty where a full combo on any of the maps is becoming challenging for some and impossible for many. We saw a handful of FCs on [HD3](https://osu.ppy.sh/beatmapsets/2198816#fruits/4653147), [HR1](https://osu.ppy.sh/beatmapsets/2198791#fruits/4653099) and [HR3](https://osu.ppy.sh/beatmapsets/2198812#fruits/4653138).

The most picked maps in the pool this week are [NM4](https://osu.ppy.sh/beatmapsets/2198872#fruits/4653265) and [MM2](https://osu.ppy.sh/beatmapsets/2198798#fruits/4653117) whilst [NM1](https://osu.ppy.sh/beatmapsets/2198766#fruits/4653068) and [DT2](https://osu.ppy.sh/beatmapsets/2198875#fruits/4653272) were never picked.

> The mappool selection team is treating the players this year with yet another [osu! original](wiki/osu!_originals) coming out for this mappool. The Mixed Mod 3 slot is filled by [Jemzuu](https://osu.ppy.sh/users/7890134)'s [Neverlast](https://osu.ppy.sh/beatmapsets/2198806#fruits/4653125), a song produced by [Aoi](https://osu.ppy.sh/beatmaps/artists/183).

Reviewer: ```suggestion
The mappool selection team is treating the players this year with yet another [osu! original](/wiki/osu!_originals) coming out for this mappool. The Mixed Mod 3 slot is filled by [Jemzuu](https://osu.ppy.sh/users/7890134)'s [Neverlast](https://osu.ppy.sh/beatmapsets/2198806#fruits/4653125), a song produced by [Aoi](https://osu.ppy.sh/beatmaps/artists/183).
```

[![](/wiki/shared/news/2024-06-27-osu-catch-world-cup-2024-finals-recap/neverlast.jpg)](https://osu.ppy.sh/beatmapsets/2198806#fruits/4653125)

<div class="osu-md__paragraph">
    <audio controls>
        <source src="https://assets.ppy.sh/artists/183/Songs/Aoi_-_Neverlast.mp3">
    </audio>
</div>

## Finals results

> We started the weekend on saturday with two loser bracket matches, where Poland, Chile, Finland and Canada fought it out to advance one match futher and secure atleast a top 4 finish this year. 

Reviewer: just remove the extra space at the end of the sentence as well

> On sunday, Argentina and the United States faced off in the winners finals. The United States have only lost a single map in the entire tournament up until now. Argentina on the other hand has had multiple close encounters and made it to winners finals being the underdogs in most of their recent matches.

Reviewer: Sunday should be capitalized no?

> The weekend ended with the winners on saturday battling it out to reach the grand finals.

Reviewer: same as above, the day should be capitalized I think

| Team A |  |  | Team B |
| --: | :-: | :-: | :-- |
| Chile ::{ flag=CL }:: | 6 | **7** | ::{ flag=PL }:: **Poland** |
| Finland ::{ flag=FI }:: | 1 | **7** | ::{ flag=CA }:: **Canada** |
| **United States** ::{ flag=US }:: | **7** | 1 | ::{ flag=AR }:: Argentina |
| Canada ::{ flag=CA }:: | 5 | **7** | ::{ flag=PL }:: **Poland** |

## Match highlights

### ::{ flag=CL }:: Chile vs. ::{ flag=PL }:: Poland

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2179700308&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
</div>

> Both Chile and Poland have looked very strong during this years osu!catch world cup, so the stage was set and the expectations high as these two titans faced off to secure themselves a top 4 finish. Chile is the final remaining country that has previously won the world cup, albeit this being 10 years ago already.

Reviewer: I think it's fine to use the abbreviation at this point since "osu!catch World Cup" was used like 2-3 times already previously in this text
```suggestion
Both Chile and Poland have looked very strong during this year's CWC, so the stage was set and the expectations high as these two titans faced off to secure themselves a top 4 finish. Chile is the final remaining country that has previously won the world cup, albeit this being 10 years ago already.
```

> Both teams won their first two picks. On Chile's third pick, however, Poland were able to grab their first breakpoint of the match. [trig0n](https://osu.ppy.sh/users/3704228) led his team towards a very narrow 7.000 points victory on [DT4](https://osu.ppy.sh/beatmapsets/2198802#fruits/4653121). With another MVP by trig0n Poland won their next pick and got into a comfortable 4-2 lead in this BO13 match versus Chile.

Reviewer: ```suggestion
Both teams won their first two picks. On Chile's third pick however, Poland were able to grab their first breakpoint of the match. ::{ flag=PL }:: [trig0n](https://osu.ppy.sh/users/3704228) led his team towards a very narrow 7,000 points victory on [DT4](https://osu.ppy.sh/beatmapsets/2198802#fruits/4653121). With another MVP by ::{ flag=PL }:: trig0n, Poland won their next pick and got into a comfortable 4–2 lead in this BO13 match versus Chile.
```

> Both teams won their first two picks. On Chile's third pick however, Poland were able to grab their first breakpoint of the match. ::{ flag=PL }:: [trig0n](https://osu.ppy.sh/users/3704228) led his team towards a very narrow 7,000 points victory on [DT4](https://osu.ppy.sh/beatmapsets/2198802#fruits/4653121). With another MVP by ::{ flag=PL }:: trig0n, Poland won their next pick and got into a comfortable 4–2 lead in this BO13 match versus Chile.

Reviewer: ```suggestion
Both teams won their first two picks. On Chile's third pick however, Poland were able to grab their first breakpoint of the match. ::{ flag=PL }:: [trig0n](https://osu.ppy.sh/users/3704228) led his team towards a very narrow 7,000 points victory on [DT4](https://osu.ppy.sh/beatmapsets/2198802#fruits/4653121). With another MVP by ::{ flag=PL }:: [trig0n](https://osu.ppy.sh/users/3704228), Poland won their next pick and got into a comfortable 4–2 lead in this BO13 match versus Chile.
```

> To make matters worse for the Chilean team, Poland was able to break them a second time. This time, [Phob]() led his team to a 5-2 advantage on [NM4](https://osu.ppy.sh/beatmapsets/2198872#fruits/4653265). Backs against the wall, Chile broke back immediately with a very solid Hidden score coming out from rookie [Thaemine](https://osu.ppy.sh/users/32442896) on [MX3](https://osu.ppy.sh/beatmapsets/2198806#fruits/4653125). On the very next map, [HR3](https://osu.ppy.sh/beatmapsets/2198812#fruits/4653138), [Thaemine](https://osu.ppy.sh/users/32442896) produced a brilliant full-combo cutting down Polands lead to only a single point.

Reviewer: ```suggestion
To make matters worse for the Chilean team, Poland was able to break them a second time. This time, ::{ flag=PL }:: [Phob](https://osu.ppy.sh/users/6069462) led his team to a 5–2 advantage on [NM4](https://osu.ppy.sh/beatmapsets/2198872#fruits/4653265). Backs against the wall, Chile broke back immediately with a very solid Hidden score coming out from rookie ::{ flag=CL }:: [Thaemine](https://osu.ppy.sh/users/32442896) on [MX3](https://osu.ppy.sh/beatmapsets/2198806#fruits/4653125). On the very next map, [HR3](https://osu.ppy.sh/beatmapsets/2198812#fruits/4653138), ::{ flag=PL }:: [Thaemine](https://osu.ppy.sh/users/32442896) produced a brilliant full-combo cutting down Poland's lead to only a single point.
```

> Both teams won their next picks: Poland won [HD2](https://osu.ppy.sh/beatmapsets/2198958#fruits/4653546) convincingly and Chile won [HR1](https://osu.ppy.sh/beatmapsets/2198791#fruits/4653099) on the back of an impressive full-combo from [Aryssiel](https://osu.ppy.sh/users/2389481). Poland is leading 6-5 and have the final pick to their disposal as well. There aren't many potential maps left in the pool and they make the wrong choice going for [NM3](https://osu.ppy.sh/beatmapsets/2198988#fruits/4653607). Chile secures a 6th point and give us the tiebreaker this match deserves! 

Reviewer: ```suggestion
Both teams won their next picks — Poland won [HD2](https://osu.ppy.sh/beatmapsets/2198958#fruits/4653546) convincingly and Chile won [HR1](https://osu.ppy.sh/beatmapsets/2198791#fruits/4653099) on the back of an impressive full combo from ::{ flag=CL }:: [Aryssiel](https://osu.ppy.sh/users/2389481). Poland is leading 6–5 and have the final pick to their disposal as well. There aren't many potential maps left in the pool, and they make the wrong choice going for [NM3](https://osu.ppy.sh/beatmapsets/2198988#fruits/4653607). Chile secures a 6th point and give us the tiebreaker this match deserves!
```

> The pressure on both teams is immense as a top 4 finish is on the line here. However, there is one man that steps up to the task. [trig0n](https://osu.ppy.sh/users/3704228), the number one in Poland, lives up to his name and absolutely outperforms the entire lobby with a 3-miss run on the [TB](https://osu.ppy.sh/beatmapsets/2198809#fruits/4653132). His astonishing performance helps secure Poland their first top 4 finish on the osu!catch world cup stage.

Reviewer: ```suggestion
The pressure on both teams is immense as a top 4 finish is on the line here. However, there is one man that steps up to the task. ::{ flag=PL }:: [trig0n](https://osu.ppy.sh/users/3704228), the number one in Poland, lives up to his name and absolutely outperforms the entire lobby with a 3-miss run on the [TB](https://osu.ppy.sh/beatmapsets/2198809#fruits/4653132). His astonishing performance helps secure Poland their first top 4 finish on the CWC stage.
```

### ::{ flag=CA }:: Canada vs. ::{ flag=PL }:: Poland

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2179927104&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
</div>

Canada convincingly won their match against Finland and also secured themselves top 4. As a reward, they get to battle it out with Poland which for both teams would be their first podium finish ever.

> This match started very similarly to the match between Chile and Poland, as both Canada and Poland won their first three picks. [trig0n](https://osu.ppy.sh/users/3704228) matched [Aryssiels](https://osu.ppy.sh/users/2389481) insane full-combo on [HR1](https://osu.ppy.sh/beatmapsets/2198791#fruits/4653099) and [Phob](https://osu.ppy.sh/users/6069462) got an impressive one-miss on [HR2](https://osu.ppy.sh/beatmapsets/2198874#fruits/4653271).

Reviewer: Arrysiel? Is this supposed to be matching like the best score on the map period, or did you put the wrong player here?
```suggestion
This match started very similarly to the match between Chile and Poland, as both Canada and Poland won their first three picks. ::{ flag=PL }:: [trig0n](https://osu.ppy.sh/users/3704228) matched ::{ flag=CL }:: [Aryssiels](https://osu.ppy.sh/users/2389481) insane full-combo on [HR1](https://osu.ppy.sh/beatmapsets/2198791#fruits/4653099), and ::{ flag=PL }:: [Phob](https://osu.ppy.sh/users/6069462) got an impressive one-miss on [HR2](https://osu.ppy.sh/beatmapsets/2198874#fruits/4653271).
```

Author: Aryssiel FCd the map the day before, now trig0n FCd today. Might be a bit too ambigious wording

> At a 3-3 scoreline, underdogs Canada actually broke team Poland on [HD2](https://osu.ppy.sh/beatmapsets/2198958#fruits/4653546). They also won their next pick [HR3](https://osu.ppy.sh/beatmapsets/2198812#fruits/4653138) and went up 5-3. However, Poland won the next two picks on [MX3](https://osu.ppy.sh/beatmapsets/2198806#fruits/4653125) and [DT3](https://osu.ppy.sh/beatmapsets/2198800#fruits/4653119) to tie things up at 5 a piece. 

Reviewer: The first sentence can be worded better, specifically the underdogs part but I can't think of anything right now
```suggestion
At a 3–3 scoreline, underdogs Canada actually broke team Poland on [HD2](https://osu.ppy.sh/beatmapsets/2198958#fruits/4653546). They also won their next pick [HR3](https://osu.ppy.sh/beatmapsets/2198812#fruits/4653138) and went up 5–3. However, Poland won the next two picks on [MX3](https://osu.ppy.sh/beatmapsets/2198806#fruits/4653125) and [DT3](https://osu.ppy.sh/beatmapsets/2198800#fruits/4653119) to tie things up at 5 a piece.
```

Reviewer 2: ```suggestion
Tied at 3 points, Canada, the underdogs, actually got a break point against team Poland on [HD2](https://osu.ppy.sh/beatmapsets/2198958#fruits/4653546). They also won their next pick [HR3](https://osu.ppy.sh/beatmapsets/2198812#fruits/4653138) and went up 5–3. However, Poland won the next two picks on [MX3](https://osu.ppy.sh/beatmapsets/2198806#fruits/4653125) and [DT3](https://osu.ppy.sh/beatmapsets/2198800#fruits/4653119) to tie things up at 5 a piece.
```

> There are two picks left - one for each team. Poland opted to go for the convert [NM4](https://osu.ppy.sh/beatmapsets/2198872#fruits/4653265) and very convingly win this one with a score margin of over 200.000 points. Canada, in similar fashion as Poland the day before, picked [NM3](https://osu.ppy.sh/beatmapsets/2198988#fruits/4653607). This, in the exact same turn of events, turned out to be the wrong pick as Poland this time around won it. This secures them the win, their first podium and a spot in the osu!catch history books.

Reviewer: ```suggestion
There are two picks left — one for each team. Poland opted to go for the convert [NM4](https://osu.ppy.sh/beatmapsets/2198872#fruits/4653265) and very convingly won this one with a score margin of over 200,000 points. Canada, in similar fashion as Poland the day before, picked [NM3](https://osu.ppy.sh/beatmapsets/2198988#fruits/4653607). This, in the exact same turn of events, turned out to be the wrong pick as this time around Poland won it. This secures them the win, their first podium, and a spot in the osu!catch history books.
```

---

> Join us, this weekend, for the Grand Finals on the [osulive Twitch channel](https://www.twitch.tv/osulive) to see which team takes it all as we crown a new champion! You can find all the details on the [wiki page](/wiki/Tournaments/CWC/2024).
> 
> The mappool from the Grand Finals was [showcased to the players at the end of last week](https://www.twitch.tv/videos/2179927105). There are two osu! originals including the [Grand Finals tiebreaker](https://osu.ppy.sh/beatmapsets/2202823#fruits/4662299) shown below. You can view the reveal videos for all original tracks on the [osu!archive YouTube channel](https://www.youtube.com/@osugamearchive).
> 
> <div align="center" class="osu-md__paragraph">
>     <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/bBPEdmfYPWM" frameborder="0" allowfullscreen></iframe>
> </div>

Reviewer: I would swap these two around, thoughts?
```suggestion
The mappool from the Grand Finals was [showcased to the players at the end of last week](https://www.twitch.tv/videos/2179927105). There are two osu! originals including the [Grand Finals tiebreaker](https://osu.ppy.sh/beatmapsets/2202823#fruits/4662299) shown below. You can view the reveal videos for all original tracks on the [osu!archive YouTube channel](https://www.youtube.com/@osugamearchive).

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/bBPEdmfYPWM" frameborder="0" allowfullscreen></iframe>
</div>

Join us, this weekend, for the Grand Finals on the [osulive Twitch channel](https://www.twitch.tv/osulive) to see which team takes it all as we crown a new champion! You can find all the details and schedules on the [wiki page](/wiki/Tournaments/CWC/2024).
```
